### PR TITLE
Refactored code in compliance with gdscript coding style

### PR DESCRIPTION
**Pull request by Poostaq**
Also, deleted some unnecessary file.

### CODE REVIEW

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 1]
```diff
@@ -37,84 +36,88 @@ func _set_image(path):
...
 			n.queue_free()
 	for trait in traitList:
 		if trait["TraitName"] == "Wszechstronność do kwadratu":
-			var traitButton = createTraitButton(alternateTraitButtonScene, trait)
-			var traitButtonTraitList = traitButton.get_node("MarginContainer/VBoxContainer/OptionButton")
-			fillTraitButtonTraitList(traitButtonTraitList)
-			traitButton.secondary_trait = traitButton.option_button.get_item_text(0)
-			traitButton.get_node(".").set_button_group(traitGroup)
+			var trait_button = create_trait_button(alternate_trait_button_scene, trait)
+			var traitButtonTraitList = trait_button.get_node("MarginContainer/VBoxContainer/OptionButton")
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:52]: shoud be: `var trait_button_trait_list`

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 2]
```diff
@@ -37,84 +36,88 @@ func _set_image(path):
...
 
-func createTraitButton(traitTemplate, traitData):
-	var traitButton = traitTemplate.instance()
-	traitContainer.add_child(traitButton)
-	var traitName = traitButton.get_node("MarginContainer/VBoxContainer/TraitNameLabel")
+
+func create_trait_button(traitTemplate, traitData):
+	var trait_button = traitTemplate.instance()
+	trait_container.add_child(trait_button)
+	var traitName = trait_button.get_node("MarginContainer/VBoxContainer/TraitNameLabel")
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:65]: forgot to change

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 3]
```diff
@@ -37,84 +36,88 @@ func _set_image(path):
...
-			traitButton.get_node(".").set_button_group(traitGroup)
+			var trait_button = create_trait_button(trait_button_scene, trait)
+			trait_button.get_node(".").set_button_group(traitGroup)
 
-func createTraitButton(traitTemplate, traitData):
-	var traitButton = traitTemplate.instance()
-	traitContainer.add_child(traitButton)
-	var traitName = traitButton.get_node("MarginContainer/VBoxContainer/TraitNameLabel")
+
+func create_trait_button(traitTemplate, traitData):
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:62]: forgot to change

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 4]
```diff
@@ -37,84 +36,88 @@ func _set_image(path):
...
+	var trait_button = traitTemplate.instance()
+	trait_container.add_child(trait_button)
+	var traitName = trait_button.get_node("MarginContainer/VBoxContainer/TraitNameLabel")
 	traitName.bbcode_text = "[center]%s[/center]" % traitData["TraitName"]
-	var traitDescription = traitButton.get_node("MarginContainer/VBoxContainer/TraitDescriptionLabel")
+	var traitDescription = trait_button.get_node("MarginContainer/VBoxContainer/TraitDescriptionLabel")
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:67]: forgot to change

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 5]
```diff
@@ -37,84 +36,88 @@ func _set_image(path):
...
-	else:
-		current_ethnicity += 1
-	load_ethnicity(ethnicities[current_ethnicity])
-	fillAttributeBonus(ethnicities[current_ethnicity]["Attribute"])
 
 func _on_Trait_Button_button_pressed(button):
 	var bonusAttribute
 	if ethnicities[current_ethnicity]["Name"] =="Nie twój zasrany interes":
-		bonusAttribute = attributeSelector.selected
+		bonusAttribute = attribute_selector.selected
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:81]: forgot to change

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 6]
```diff
@@ -37,84 +36,88 @@ func _set_image(path):
...
 	else:
 		bonusAttribute = ethnicities[current_ethnicity]["Attribute"]
 	emit_signal("ethnicity_chosen", button, ethnicities[current_ethnicity], bonusAttribute)
 
-func fillTraitButtonTraitList(traitListElement: OptionButton):
+
+func fill_trait_button_trait_list(traitListElement: OptionButton):
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:87]: forgot to change

File: game/EthnicityPage/EthnicityTrait.gd [hunk 7]
```diff
@@ -1,18 +1,16 @@
...
-onready var TraitNameLabel = $MarginContainer/VBoxContainer/TraitNameLabel
-signal traitButtonPressed(button)
-
+signal trait_button_pressed(button)
 export var ID: String
-export var traitName: String
+export var trait_name: String
 export var description: String
 
+onready var TraitNameLabel = $MarginContainer/VBoxContainer/TraitNameLabel
```
Comment (Malynna) [inline game/EthnicityPage/EthnicityTrait.gd:8]: forgot to change

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 8]
```diff
@@ -37,84 +36,88 @@ func _set_image(path):
...
 	for trait in traitList:
 		if trait["TraitName"] == "Wszechstronność do kwadratu":
-			var traitButton = createTraitButton(alternateTraitButtonScene, trait)
-			var traitButtonTraitList = traitButton.get_node("MarginContainer/VBoxContainer/OptionButton")
-			fillTraitButtonTraitList(traitButtonTraitList)
-			traitButton.secondary_trait = traitButton.option_button.get_item_text(0)
-			traitButton.get_node(".").set_button_group(traitGroup)
+			var trait_button = create_trait_button(alternate_trait_button_scene, trait)
+			var traitButtonTraitList = trait_button.get_node("MarginContainer/VBoxContainer/OptionButton")
+			fill_trait_button_trait_list(traitButtonTraitList)
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:53]: forgot to change attribute?

File: game/EthnicityPage/ChooseEthnicityStep.gd [hunk 9]
```diff
@@ -1,33 +1,32 @@
 extends Control
 
+signal ethnicity_chosen(traitElement, currentEthnicity, bonusAttribute)
```
Comment (Malynna) [inline game/EthnicityPage/ChooseEthnicityStep.gd:3]: forgot to change attributes

**Review by Malynna — CHANGES_REQUESTED** [review]
forgot to change some names

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 1 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:52]: Fixed

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 2 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:66]: Fixed

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 3 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:63]: Fixed

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 5 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:82]: Fixed

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 6 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:94]: Fixed

File: game/EthnicityPage/EthnicityTrait.gd (hunk 7 above)
Comment (Poostaq) [reply game/EthnicityPage/EthnicityTrait.gd:8]: Fixed

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 9 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:3]: Fixed

File: game/EthnicityPage/ChooseEthnicityStep.gd (hunk 8 above)
Comment (Poostaq) [reply game/EthnicityPage/ChooseEthnicityStep.gd:54]: Fixed

**Review by Malynna — APPROVED** [review]

**Pull request merged**